base/applications/sc/config.c

```diff
@@ -17,7 +17,7 @@ BOOL QueryConfig(LPCTSTR ServiceName)
     DWORD cbBytesNeeded = 0;
     LPQUERY_SERVICE_CONFIG pServiceConfig = NULL;
     LPWSTR lpPtr;
-    INT nLen, i;
+    SSIZE_T nLen, i;
```

Comment:
Do we really need to keep these signed?

Reply:
Probably not. I just did it this way to avoid signed vs unsigned issues. Remember this is all about making 64 bits great again, not about fixing unrelated stuff.

Reply:
Probably not, but I don't intend to change it., since it is unrelated and can potentially cause issues.